go/arrow/Gopkg.toml

```diff
@@ -17,3 +17,7 @@
 [[constraint]]
   name = "github.com/stretchr/testify"
   version = "1.2.0"
+
+[[constraint]]
+  name = "github.com/pkg/errors"
```

Comment:
could you also update the `go.mod` and `go.sum` files?

Reply:
you'd just need to go to `arrow/go/arrow` and do:
```
$> GO111MODULE=on go get -v ./...
```

that should do the trick.